fmriprep/cli/run.py

```diff
@@ -79,10 +79,13 @@ def get_parser():
                              'default to compensate for field inhomogeneities.')
     g_conf.add_argument(
         '--output-space', required=False, action='store',
-        choices=['T1w', 'MNI152NLin2009cAsym',
-                 'fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'],
-        nargs='+', default=['MNI152NLin2009cAsym', 'fsaverage5'],
+        choices=['T1w', 'template', 'fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'],
+        nargs='+', default=['template', 'fsaverage5'],
         help='volume and surface spaces to resample functional series into')
```

Comment:
Sorry I did not catch this before - could you briefly describe what 'template' means in this help string and refer people to `--template`?